frontend/src/redux/store/index.tsx

```diff
@@ -1,7 +1,11 @@
 import { createStore } from 'redux'
 import reducer from '../reducers';
 
-const store = createStore(reducer)
+const store = createStore(
+    reducer,
+    (window as any).__REDUX_DEVTOOLS_EXTENSION__ &&
+    (window as any).__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
Keeping this always, so that we can debug with redux-devtools in prod as well. Would be super-useful if there's a bug.